assets/css/editor.scss

```diff
@@ -28,6 +35,43 @@
 		display: block;
 	}
 
+	.star-rating {
```

Comment:
This is fine for now, but I raised the point here https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/576#issuecomment-494338138

Reply:
Yeah, if we go with your "option 1", we'll end up moving this to style.css, but we can handle that then 👍